### PR TITLE
Karam dosa

### PR DESCRIPTION
**Pull request by Jaikrishna7271**

### CODE REVIEW

File: menu.txt [hunk 1]
```diff
@@ -1 +1,3 @@
 dosa
+karam
+oil
```
Comment (jkrishna8919) [inline menu.txt:3]: why u add oil at the end?

Comment (Jaikrishna7271) [reply menu.txt:3]: extra oil

**Review by jkrishna8919 — APPROVED** [review]
everything is good go ahead

**Comment by Jaikrishna7271** [comment]
good to go

**Pull request merged**